internal/pkg/storage/runtime_test.go

```diff
@@ -621,10 +629,13 @@ var _ = t.Describe("Runtime", func() {
 		It("should fail to create a container on invalid image ID", func() {
 			// Given
 			// When
+			inOrder(
```

Comment:
I think these additional inOrder are not needed at all. Can you please apply this commit to fix the mock oder:
https://github.com/openSUSE/cri-o/commit/5c4b09d53611acb9ac23c2cdeb65ee5a95dcc8c6

Reply:
@saschagrunert thanks! That seems to have fixed the problem. I have cherry-picked your commit in this PR.